qcfractal/interface/client.py

```diff
@@ -79,6 +80,18 @@ def __init__(self,
 
         self.server_name = self.server_info["name"]
 
+        def version_list(version):
+            re.sub(r"\+.*$", "", version.strip("v"))
```

Comment:
Does `re` update in place?

Reply:
No, typo on my part

Reply:
better to run `safe_version` on everything?

Reply:
We control both sides... I think we better convert this to a tuple and ship it that way from the server and be done with it.